contribs/gmf/src/controllers/abstract.js

```diff
@@ -180,4 +181,31 @@ gmf.AbstractController.prototype.initLanguage = function() {
   }
 };
 
+/**
+ * @param {angular.$injector} $injector Main injector.
+ * @return {olx.ViewOptions} The view options from the permalink.
+ * @protected
+ */
+gmf.AbstractController.prototype.getPermalinkViewConfig = function($injector) {
```

Comment:
Don't send `$injector` as parameter.
Either put it in `this` in the constructor, or put the `gmfPermalink` service in `this` in the constructor.